serverless.yml

```diff
@@ -64,7 +74,7 @@ resources:
         MetricName: ConcurrentExecutions
         Statistic: Maximum
         ComparisonOperator: GreaterThanOrEqualToThreshold
-        Threshold: 150
+        Threshold: 48
```

Comment:
48 is a little magical.  Why not 50?

Reply:
I guess I wanted to know a little before it hit 50. All things considered, if it reaches 48, it would probably hit 50 in short order.